scripts/ci/run_perftest.sh

```diff
@@ -273,6 +273,17 @@ query_types() {
   esac
 }
 
+query_interval() {
+  case $1 in
+    battery-levels)
+      echo -query-interval=5m
+      ;;
+    *)
+      echo ""
```

Comment:
It might be worth it here to either add an explicit default `-query-interval=<whatever you want the default to be>` here or else a comment stating what the implicit default value is. Not a blocker though.

Reply:
Good idea, I'll add a comment. I didn't want to specify a default here since that would be a duplicate/override of the default with the query generation CLI tool itself - leaving the flag blank here will allow the query gen tool to use its default, and if the default value ever needs to change it will only need to change in that one place.